js/app.js

```diff
@@ -120,3 +120,70 @@ popDeactivators.forEach(popDeactivator => {
 document.querySelector('.pop-proj-1').addEventListener('click', e => {
   e.stopPropagation();
 });
+
+// Form Validation Action
+const textValid = document.querySelector('.textarea');
+const emailValid = document.querySelector('.email-input');
+const nameValid = document.querySelector('.name-input');
+const formBut = document.querySelector('.form-button');
+const emailTest = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+const eventBut = data => {
+  if (data.value === '') {
+    return true;
+  }
+  return false;
+};
+
+const defaultButton = value => {
+  formBut.disabled = value;
+};
+defaultButton(true);
+
+const nameValidator = () => {
+  if (nameValid.value === '') {
+    defaultButton(true);
+  } else if (eventBut(emailValid) === true || !emailTest.test(emailValid.value)) {
+    defaultButton(true);
+  } else if (eventBut(textValid) === true || textValid.value.length < 15) {
+    defaultButton(true);
+  } else {
+    defaultButton(false);
+  }
+};
+
+const emailValidator = () => {
+  if (!emailTest.test(emailValid.value) || emailValid.value === '') {
+    document.querySelector('.emailArea-note').classList.add('textarea-note-active');
+    defaultButton(true);
+  } else {
+    document.querySelector('.emailArea-note').classList.remove('textarea-note-active');
+    if (eventBut(textValid) === true || eventBut(nameValid) === true) {
+      defaultButton(true);
+    } else if (textValid.value.length < 15) {
+      defaultButton(true);
+    } else {
+      defaultButton(false);
+    }
+  }
+};
```

Comment:
- [ ] Kindly check if the email is all lowercase by comparing if the email and its lowercase converted version are the same, ie: `email === email.toLowerCase()`, utilizing regular expressions, or looping through each letter of the email string. Any implementation of this check should be considered valid as long as it works properly😎😎 in simple terms, the contact form contains at least one validation checking if the email is written in all lowercase:

* If a user submits the contact form with the email field containing uppercase characters an error message should show up.
*

Reply:
Noted boss.
I have made some modifications to the code to help enable that the email is always in lowercase.